### ISSUE TITLE
Plugin nefunguje

### ISSUE THREAD

**Issue by Jindrar** [kodi-czsk/plugin.video.dmd-czech.prima#8]
Prima změnila stránky a od 14.12. nefunguje. Prosím šlo by to nějak opravit? Díky moc

**Comment by lgreg** [comment]
No, opravit to urcite pujde, ale ty stranky jsou totalne prekopany, to je temer na prepsani celyho pluginu. Jinak to vypada, vzhledem k tomu, jak ty stranky ted (ne)funguji, ze to musi primaci jeste dost poladit, tzn. budou dalsi zmeny. Vzhledem k tomu, ze sem ted totalne zasekanej, tak pokud se k oprave nedostane nekdo driv, tak ja bych se k tomu dostal az nekdy mezi svatkama.

**Comment by alladdin** [comment]
V rychlosti jsem na to kouknul a myslim si, ze tam budou minimalne dva problemy.
Z toho to, ze ma ten web jinou strukturu je ten mensi.
Vetsi problem asi bude s videem.
Co jsem koukal, tak prehravac pouziva manifest.mpd, coz je MPEG DASH format, ktery kodi zatim nepodporuje. Asi by to slo nejak obejit a parsovat si to sam, ale neni ti na jedno odpoledne.

**Comment by mx3L** [comment]
Staci zmenit link z manifest.mpd na manifest.m3u8:

`http://se03.se.prima-vod-prep.service.cdn.cra.cz/vod_Prima/_definst_/0001/5232/cze-ao-sd1-sd2-sd3-sd4.smil/manifest.mpd`
->
`http://se03.se.prima-vod-prep.service.cdn.cra.cz/vod_Prima/_definst_/0001/5232/cze-ao-sd1-sd2-sd3-sd4.smil/manifest.m3u8`

**Comment by zemos8926** [comment]
A kde změním ten link ? Díky

**Comment by alladdin** [comment]
Tak jsem začal s přepisem addonu pro Prima Play. Pro všechny co se o tento addon zajímají, přidávám odkaz na první testovací verzi [plugin.video.primaplay 0.1.0](https://github.com/alladdin/plugin.video.primaplay/releases/tag/0.1.0)

Je to první verze, navíc můj první počin pro Kodi a ještě k tomu první věc v Pythonu. Chyby prosím hlásit, nějak se s nimi poperu;)

**Comment by Jindrar** [comment]
Super díky moc, kdyby tam šla přidat ještě možnost stáhnout video na HDD. V minulém doplňku mi to chybělo. 
Jinak mi to hlásí chybu : malformed start tag, at line 103, column 43

**Comment by lgreg** [comment]
Sry, mezi svatkama jsem se k tomu nedostal :( Super ze se k tomu dostal nekdo misto me, ja se na ten plugin mrknu a kdyztak to muzem probrat co s tim dal, muzes se mi Lado prosim ozvat na mejl? Diky

**Comment by lukasenkocz** [comment]
Dobrá práce. Vypadá to skvěle. Testováno na Kodi Raspberry pi 2.

**Comment by Liber81** [comment]
S KODI 15.2 na Raspebrry PI s Raspbianem nelze plugin.video.primaplay 0.1.0 nainstalovat. Důvod - nelze instalovat závislosti. Prosím o radu pro laika :-)

**Comment by Liber81** [comment]
Také potvrzuji funkčnost na Raspberry Pi (1) s Raspbianem a KODI 15.2.

**Comment by lzoubek** [comment]
jakym zpusobem navhrujete zaclenit novy Prima Play plugin ? https://github.com/kodi-czsk/repository/pull/12

Myslim, ze mit 2 navic se sejnym nazvem bude pro uzivatele dost matouci. 

Pokud ma novy kousek od @alladdin nahradit tento, oznacme tento jako nefukncni a nastavme ten novy, jako zavislost tohoto, coz by melo zpusobit auto instalaci.

**Comment by alladdin** [comment]
Udelal jsem jeste pull request kodi-czsk/plugin.video.dmd-czech.prima#9 do stareho pluginu

**Comment by E000001** [comment]
Sorry, ale myslím, že to takto nebude fungovat, protože tento nový doplněk:
<import addon="plugin.video.primaplay" version="0.2.1"/>

není možné nikterak naimportovat, nebude vědět odkud ho vzít, protože není součástí repozitáře KODI CZ/SK.

Navrhuji doplněk přidat do celého repozitáře KODI CZ/SK  a @alladdinovi udělat přístup pro správu tohoto jeho doplňku.

**Comment by lzoubek** [comment]
Jasne, je treba napred alladinuv kousek pridat. Nechtel jsem ho ale mergovat, dokud nebude jasne, ze tento doplnek konci svuj zivot - a jeho autor(i) s tim souhlasi. Dnes vecer provedu merge do  kodi-czsk/repository#12, a pak tento plugin updatnu ( dle #9 )

**Comment by bodyguardroman** [comment]
Funguje:) mimochodom teraz je nastavena SD kvalita,mohol by si doplnit,,ak sa to bude dat,aby sa dala nastavit HD kvalita?(ako to bolo v povodnom doplnku)

**Comment by alladdin** [comment]
Bohuzel rozliseni videa je 720x400 a neni moznost si vybrat jine ani na webu primy. Video je akorat v ryznych datovych tocich, ale to si zvoli kodi samo.

Navrhy a chyby k novemu pluginu prosim davejte na https://github.com/alladdin/plugin.video.primaplay/issues

**Comment by E000001** [comment]
@alladdin můžeme prosím zahrnout tuto tvoji verzi do repozitáře a nahradit tím tak ten starý addon? Dostalo by se to k více lidem a automaticky bez námahy :)

**Comment by kapitan-iglu** [comment]
Tiez sa prihovaram za nahradu stareho addonu novym - funkcnym. Archivovat stary nema zmysel (historia je na gite) a vznika tym len chaos (ktory addon je udrzovany, kam hlasit chyby, ktory mam vlastne nainstalovany atd...)

**Comment by E000001** [comment]
@alladdin : Tak co, jak to vidíš? bylo by opravdu lepší to spojit :)
